<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameters_Dialog</class>
 <widget class="QDialog" name="Parameters_Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>592</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>225</width>
     <height>222</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nx:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="nxSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ny:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="nySpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Nz:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="nzSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="lambdaNumSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lambda:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="lambdaDenSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99.000000000000000</double>
        </property>
        <property name="value">
         <double>29.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Vent:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="ventSpinBox">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Delta:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="deltaSpinBox">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.017000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>DeltaPrime:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="deltaPrimeSpinBox">
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Montagne 1</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>137</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Montagnes:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_4">
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>481</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Montagne pointue</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="browse_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>590</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Veuillez patienter svp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Parameters_Dialog</receiver>
   <slot>add_or_remove_mountain(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nxSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>nySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nxSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>nzSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Parameters_Dialog</receiver>
   <slot>montagne_pointu_button(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_button</sender>
   <signal>clicked()</signal>
   <receiver>Parameters_Dialog</receiver>
   <slot>browse_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_ok()</slot>
  <slot>nx_changed(int)</slot>
  <slot>ny_changed(int)</slot>
  <slot>nz_changed(int)</slot>
  <slot>lambdaden_changed(double)</slot>
  <slot>lambdanum_changed(double)</slot>
  <slot>vent_changed(double)</slot>
  <slot>delta_changed(double)</slot>
  <slot>deltaprime_changed(double)</slot>
  <slot>add_or_remove_mountain(int)</slot>
  <slot>montagne_pointu_button(bool)</slot>
  <slot>browse_files()</slot>
  <slot>waiting_time()</slot>
 </slots>
</ui>
